lib/fluent/config/element.rb

```diff
@@ -142,7 +142,7 @@ def to_s(nest = 0)
         indent = "  " * nest
         nindent = "  " * (nest + 1)
         out = ""
-        if @arg.empty?
+        if @arg.nil? or @arg.empty?
```

Comment:
[nits] I think Rubyist prefers `||` to `or`. (`and` is the same.  preferred `&&` to `and`)

```suggestion
        if @arg.nil? || @arg.empty?
```

Reply:
`@arg` seems to be expected String type. when does `@arg.nil? #=> true` happen?

Reply:
`nil` will be set by `use_nil` in embedded Ruby code which is introduced in this PR, like the following case:

```
<section>
  <subsection "#{ENV['SOME_ENV_VAR'] || use_nil}">
    ...
  </subsection>
<sectin>
```